packages/admin/admin-sdk/package.json

```diff
@@ -21,13 +21,10 @@
   "devDependencies": {
     "@types/react": "^18.3.2",
     "tsup": "^8.0.1",
-    "typescript": "^5.3.3",
-    "zod": "^3.22"
+    "typescript": "^5.3.3"
   },
   "dependencies": {
-    "@medusajs/admin-shared": "~2.4.0"
-  },
-  "peerDependencies": {
+    "@medusajs/admin-shared": "~2.4.0",
     "zod": "^3.22"
```

Comment:
I see the usage of zod is in context to validating the config and the schemas are not used inside the Medusa app. Hence, `zod` can be kept as a regular dependency.

Do let me know, if my assumption is wrong